development/generate-rc-commits.js

```diff
@@ -129,18 +57,26 @@ async function getPRLabels(owner, repo, prNumber) {
   }
 }
 
-// Function to get the team for a given author
-function getTeamForAuthor(authorName) {
-  for (const [team, authors] of Object.entries(authorTeams)) {
-    if (authors.includes(authorName)) {
-      return team;
-    }
+// Function to get the GitHub username for a given commit hash
+async function getGitHubUsername(commitHash) {
```

Comment:
In the teams.json file, the PR author's username (handle) is used, so we need to retrieve the username instead of the author's name from the commit log.